scijava/scijava-types/src/main/java/org/scijava/types/Types.java

```diff
@@ -1676,6 +1678,10 @@ private static boolean isAssignable(final Type type,
 				return true;
 			}
 
+			if (type.equals(Any.class)) {
```

Comment:
`Any` is not `final`.... so this check would fail if someone extends it.... :wink: 

(To clarify: I think `Any` should probably be final—but if not, this line should change to something like `type instanceof Class && Any.class.isAssignableFrom((Class<?>) type)`.)

Reply:
Made `Any` `final`